deployment/README.md

```diff
@@ -134,3 +134,98 @@ Run the following commands to apply the **container_file_t** SELinux label to th
 # semanage fcontext -a -t container_file_t "<STORAGE-PATH>(/.*)?"
 # restorecon -Rv <STORAGE-PATH>
 ```
+Add a new section to https://github.com/IBM/Db2/tree/develop/deployment.
+
+
+# How to create a new LDAP user
+## 1. Allocate the LDAP pod
+``` 
+oc get po | grep ldap | cut -d " " -f 1
+```
+Example output:
+```
+db2u-deployment-db2u-ldap-5cf745f7cf-ql6sh
+```
+## 2. Log in to the LDAP pod
+```
+oc rsh <LDAP-pod-name> /bin/bash
+```
+## 3. Find the next available LDAP user ID
+```
+source /opt/ibm/lib/utils.sh
+
+ldapPassword=$(get_ldap_root_password)
+
+echo $(($(ldapsearch -Z -H ldap:/// -D 'cn=bluldap,dc=blustratus,dc=com' -w "$ldapPassword" -b 'dc=blustratus,dc=com' '(objectClass=posixAccount)' | grep uidNumber | cut -d ":" -f 2 | sort -nr | head -n 1)+1)) 
```

Comment:
Could we have a variable like LDAP_UID=....
Then, we only have to reference the variable in the script

Reply:
In this case, the UID value needs to be manually edited inside the config file newuser.ldif. 

We plan to fully automate the setup (including auto generate the config file, etc) in the future.